ui/selectors/selectors.js

```diff
@@ -1091,6 +1091,9 @@ export function getSortedAnnouncementsToShow(state) {
   return announcementsSortedByDate;
 }
 
+export function getOrderedNetworksList(state) {
```

Comment:
Could you add this to the `metamask` slice instead? This selectors file is meant more for cross-slice selectors

Reply:
I did not understand this. What do you mean by metamask slice here?